e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js

```diff
@@ -124,7 +124,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
         });
     });
 
-    it('Post Reactions option removed for Guests and Members in System Scheme', () => {
+    it('MM-T1545 Post Reactions option removed for Guests and Members in System Scheme', () => {
```

Comment:
Same here. It has 4 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

Reply:
Consolidated steps in TM4J